src/v2/components/CustomBadgeUploader/index.js

```diff
@@ -4,20 +4,18 @@ import { Query } from 'react-apollo'
 import customBadgeQuery from 'v2/components/CustomBadgeUploader/queries/customBadge'
 
 import Uploader from 'v2/components/CustomBadgeUploader/components/Uploader'
-import ErrorAlert from 'v2/components/UI/ErrorAlert'
-import LoadingIndicator from 'v2/components/UI/LoadingIndicator'
 
 export default class CustomBadgeUploader extends PureComponent {
   render() {
     return (
       <Query query={customBadgeQuery} ssr={false}>
         {({ loading, error, data, startPolling, stopPolling }) => {
           if (loading) {
-            return <LoadingIndicator p={6} />
+            return <span />
           }
 
           if (error) {
-            return <ErrorAlert m={6}>{error.message}</ErrorAlert>
+            return <span />
```

Comment:
?

Reply:
In reality, showing an error here doesn’t really do anything for the user, but we also don’t want to render the component.

Reply:
return null?